modules/mediasquareBidAdapter.js

```diff
@@ -109,25 +113,30 @@ export const spec = {
      * @param {ServerResponse[]} serverResponses List of server's responses.
      * @return {UserSync[]} The user syncs which should be dropped.
      */
-  getUserSyncs: function(syncOptions, serverResponses, gdprConsent) {
+  getUserSyncs: function(syncOptions, serverResponses, gdprConsent, uspConsent) {
     let params = '';
     let endpoint = document.location.search.match(/msq_test=true/) ? BIDDER_URL_TEST : BIDDER_URL_PROD;
-    if (gdprConsent && typeof gdprConsent.consentString === 'string') {
-      if (typeof gdprConsent.gdprApplies === 'boolean') { params += `&gdpr=${Number(gdprConsent.gdprApplies)}&gdpr_consent=${gdprConsent.consentString}`; } else { params += `&gdpr_consent=${gdprConsent.consentString}`; }
-    }
-    if (syncOptions.iframeEnabled) {
-      return {
-        type: 'iframe',
-        url: endpoint + BIDDER_ENDPOINT_SYNC + '?type=iframe' + params
-      };
-    }
-    if (syncOptions.pixelEnabled) {
-      return {
-        type: 'image',
-        url: endpoint + BIDDER_ENDPOINT_SYNC + '?type=pixel' + params
-      };
+    if (serverResponses[0].body.hasOwnProperty('cookies') && typeof serverResponses[0].body.cookies === 'object') {
+      return serverResponses[0].body.cookies;
+    } else {
+      if (gdprConsent && typeof gdprConsent.consentString === 'string') {
+        if (typeof gdprConsent.gdprApplies === 'boolean') { params += `&gdpr=${Number(gdprConsent.gdprApplies)}&gdpr_consent=${gdprConsent.consentString}`; } else { params += `&gdpr_consent=${gdprConsent.consentString}`; }
```

Comment:
Instead of this 1-line if-else, use the ternary operator:

```
params += typeof gdprConsent.gdprApplies === 'boolean' ? <with gdprApplies> : <without gdprApplies>
```

Reply:
sure, it's changed